### ISSUE TITLE
Angrylion plugin not working on android(Mupen64plus core)

### ISSUE THREAD

**Issue by mckimiaklopa** [libretro/RetroArch#1222]
I dont really know what wrong Ive been doing.I have set the rsp to cxd4 but the angrylion plugin is just a black scree.

My apk is an October 30 build

**Comment by mckimiaklopa** [comment]
still does not work on my android device(feb. nightly build of mupen64plus)
device:Galaxy s Duos 2 with android Jb

**Comment by mckimiaklopa** [comment]
Still not working in 2016

**Comment by fr500** [comment]
even if it worked it would be unusable on your device.
Angrylion is very CPU intensive and it's definitely not suitable for a mid-range android device.

**Comment by twinaphex** [comment]
You probably did not set the rsp plugin to cxd4, it would not run with rsp hle.

Anyway, don't even bother, your android device is way too weak to even run it beyond the single diti framerate.

**Comment by mckimiaklopa** [comment]
I did set set rsp the rsp to cxd4. Mupen64 crashes upon loading a game when the plugin is set to  the angrylion one.

Of course I am fully aware of the notorious  slow speed of the angrylion plug in. I of course never intended to use it for casually playing. I intended to use it  to see the actual graphics of an n64 game.
If my netbook with overall weaker performance compared to my phone can run the angrylion plugin perfectly well albeit framrate issues aside, why am i discouraged from complaining about something that's broken on the android version?
